internal/app/kdk/up.go

```diff
@@ -20,20 +20,24 @@ import (
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
 	"github.com/docker/docker/client"
 	"github.com/docker/go-connections/nat"
 	"github.com/spf13/viper"
 )
 
 func Up(ctx context.Context, dockerClient *client.Client, imageCoordinates string, logger logrus.Entry) error {
-	var binds []string
+	var volumes map[string]struct{}
```

Comment:
This should be made the direct type: https://github.com/moby/moby/blob/master/api/types/volume

Reply:
Oh maybe not
https://github.com/moby/moby/blob/master/api/types/container/config.go#L59

Reply:
Curious why it shouldn't be:
```
        volumes := map[string]volume.Volume{}
```
Instead of
```
	var volumes map[string]struct{}
```
@drekle